distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -2015,10 +1798,17 @@ public void testCancelDownload() throws Exception {
                 "Resource retrieval cancelled. ",
```

Comment:
We should use the last notification message like we do for activities instead of hard-coding `0` on line 1796.